index.html

```diff
@@ -15,27 +15,33 @@
 
 <body>
 
+<!-- ─── Desktop Header ─── -->
+
 <!-- ─── Header ─── -->
 
 <header class="header">    
     <nav class="navbar">
         <div class="logo">
             <a href="#">My Logo</a>
         </div>
+        <ul>
+            <li><a class="navigation" href="#works">Portfolio</a></li>
+            <li><a class="navigation" href="#aboutme">About</a></li>
+            <li><a class="navigation" href="#contact">Contact</a></li>
+        </ul>
         <div class="hamburger">
             <div class="bar top"></div>
             <div class="bar middle"></div>
             <div class="bar bottom"></div>
           </div>
-            </button>
-              </nav>
-    </header>
+    </nav>
+</header>
 
 <!-- ─── Headline ─── -->
 
         <section class="headline">
             <div class="text">
-                <h1>I’m Nurbol<br>Glad to see<br>you!</h1>
+                <h1>I’m Nurbol<br>Glad to see you!</h1>
```

Comment:
- [ ] Everything is good the only small change you need to make is to adjust the spacing please fix the spacing as there is more spacing on the first page. See the image below for reference.
### Your design
![his](https://user-images.githubusercontent.com/81727988/214903875-a195560f-97b7-461b-82f2-2d73942c18b3.jpg)

![hisss](https://user-images.githubusercontent.com/81727988/214903907-567ee006-a45a-49a5-ad1e-baa51e347881.jpg)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/214903941-50ed2079-1ebd-4a31-9cdf-9b3b3164a4c7.jpg)